lib/inky/inkycommands.ex

```diff
@@ -144,14 +136,15 @@ defmodule Inky.Commands do
     send_command(pins, 0x32, luts)
   end
 
-  defp set_dimensions(pins, columns, packed_height) do
-    # TODO create binaries more neatly
+  defp set_dimensions(pins, display) do
```

Comment:
Largely the same considerations as for set_gate above.

Reply:
Hm... Agreed.